ex_feels/lib/ex_feels_web/templates/home/home.html.eex

```diff
@@ -1,3 +1,23 @@
+<nav class="navbar is-dark" style="bottom-margin: 50px;">
+  <div class="navbar-brand level-item has-text-centered">
+    <img src="/images/brain.jpg" alt="Wojak" height="100px">
```

Comment:
I like where you are going with this but the resolution is lacking for this particular asset.